stratum/hal/lib/common/utils.cc

```diff
@@ -396,5 +397,18 @@ std::string ConvertHwStateToPresentString(const HwState& hw_state) {
   }
 }
 
+float Decimal64ValueToFloat(const ::gnmi::Decimal64 &value) {
+  return value.digits() / std::pow(10, value.precision());
+}
+
+::gnmi::Decimal64 FloatToDecimal64Value(
+  float value, ::google::protobuf::uint32 precision) {
+  ::gnmi::Decimal64 decimal;
+  decimal.set_digits(
+      static_cast<::google::protobuf::int64>(value * std::pow(10, precision)));
```

Comment:
Do we need the cast here or is implicit conversion enough?

Reply:
no need to use this is we are using llround